dataactcore/migrations/versions/177d90a225dc_sf_133_amount_text.py

```diff
@@ -0,0 +1,88 @@
+"""sf_133_amount_text
```

Comment:
Whoops, thought I had removed this.  Alembic autogenerate didn't notice the change in column type, so I just changed the migration that creates the table to create as text.